wandb/sdk/launch/runner/local_container.py

```diff
@@ -108,12 +108,18 @@ def run(
             if not docker_image_exists(image_uri):
                 pull_docker_image(image_uri)
             env_vars.pop("WANDB_RUN_ID")
-            # if they've given an override to the entrypoint
-            entry_cmd = get_entry_point_command(
-                entry_point, launch_project.override_args
-            )
+            entry_cmd = []
+            if entry_point is not None:
```

Comment:
By saying `is not None` we allow users to enter `entry_point=""` which may or may not be desirable, just a note.

Reply:
Well if we accept "" as a custom entrypoint, then the run has to fail right? I feel like we should error or revert to the default in that case.

Reply:
this specific instance of `entry_point` is actually an Optional `EntryPoint` class as implemented within 
 _project_spec.py. We should potentially add entrypoint validation to that class IMO